toponetx/classes/path.py

```diff
@@ -0,0 +1,120 @@
+"""Path class."""
+
+from collections.abc import Collection, Hashable
+from typing import Any, List, Tuple, Union
```

Comment:
Please don't use the deprecated `typing.List`, `typing.Tuple`, and `typing.Union` classes. TopoNetX is on Python 3.10+, so you can use the builtin `list` and `tuple` classes support subscripting and union type expression (`A | B` instead of `Union[A, B]`) is available.

Reply:
Fixed!